policies/codingstyle-updates.txt

```diff
@@ -0,0 +1,70 @@
+A summary of the discussion thread so far.  Not surprisingly, it's all about
+the whitespace. :)
+
+The descriptions here were written to be understandable stand-alone.  Once we
+come to a conclusion, we'll wordsmith them into the coding style.
+
+Do not put a space after sizeof; do use parens:
+
+	sizeof(foo)
+
+When breaking long lines, if there are Boolean conditionals, put them at the
+start of the line.  Consider doing this consistently, and don't merge even if
+they fit.  Here is a bad example, it should be three lines:
+
+	some_long_condition()
+	&& short1() && short2()
+
+Related conditions should be on one line if they fit, else given an extra
+level of indent
+
+	some_long_condition()
+	    && (short1() || short2())
+	    && some_other_flag != 0
+
+If the expression for an if statement does not fit, indent the continuation
+lines an extra level (this is not necessary for an else-if)
+
+	if (this_is_true()
+		&& !that_is_false()) {
+	    code();
+	    ...

```

Comment:
This i a bit confusing. First, you talk about giving continuation lines in conditionals *in general* an extra indent, and then you seem to give `if` a special case...

Side note: if someone has a c-mode (emacs) hack to get that extra indentation, let me know. Doing it by hand is a royal pain...

Reply:
If you have a better way to word it, please let me know.  if *is* a special case.

Reply:
I'm with Richard in sense that it *is* confusing. If extra indentation in multi-line condition is specific to "if", then having example in paragraph prior mentioning "if" is confusing.

Reply:
Just in case to clarify. It's suggested to use extra indentation [in multi-line condition] specifically with "if" [and not for example with "while" as no visual dissonance].

Reply:
Pushed commit that says "*two* indents for a continued if"  Does that help?